pkgs/applications/misc/nnn/default.nix

```diff
@@ -1,6 +1,9 @@
-{ lib, stdenv, fetchFromGitHub, pkg-config, ncurses, readline, conf ? null }:
+{ lib, stdenv, fetchFromGitHub, pkg-config, ncurses, readline
+, conf ? null, withIcons ? false, withNerdIcons ? false }:
 
-with lib;
+# Mutually exclusive options
+assert withIcons -> withNerdIcons == false;
+assert withNerdIcons -> withIcons == false;
```

Comment:
I think we only need one line but fine for me.

Reply:
Yeah, but this way we have better error output.